config/locales/views/moderations/en.yml

```diff
@@ -6,7 +6,7 @@ en:
       actions:
```

Comment:
I'm so sick of the `"` to `'` changes that I think my linter makes automatically. 😩 
However, there ARE legitimate changes I made in this file.